### ISSUE TITLE
Throw Exception"genTF unsupported label data type: DOUBLE_TYPE"

### ISSUE THREAD

**Issue by Echo9573** [sql-machine-learning/sqlflow#1166]
**Description**
Throw Exception "converting driver.Value type float64 ("-0.034224987") to a int64 " when select data from predictions in notebook.

**Reproduction Steps**
![image](https://user-images.githubusercontent.com/24820352/68540168-91a3e000-03c8-11ea-92bb-b7d7557126aa.png)

``` sql
%%sqlflow
SELECT *
FROM intelligence_da_dev.titanic_train
TRAIN DNNClassifier
WITH
  model.n_classes = 2,
  model.hidden_units = [10, 40, 10],
  train.epoch = 200,
  train.batch_size = 64
COLUMN age, fare, pclass_3, pclass_1, pclass_2, sex_male, sex_female, sibsp_1, sibsp_0, sibsp_others_type, parch_0, parch_1, parch_2, parch_others_type, embarked_s, embarked_c, embarked_others_type, embarked_unknown_type
LABEL survived
INTO intelligence_da_dev.my_dnn_model;
```

--------------------------------Exception------------------------------
_Rendezvous: <_Rendezvous of RPC that terminated with:
  status = StatusCode.UNKNOWN
  details = "genTF unsupported label data type: DOUBLE_TYPE"
  debug_error_string = "{"created":"@1573309027.737003066","description":"Error received from peer ipv4:10.82.128.7:8005","file":"src/core/lib/surface/call.cc","file_line":1055,"grpc_message":"genTF unsupported label data type: DOUBLE_TYPE","grpc_status":2}"
>

**Comment by Echo9573** [comment]
#770

**Comment by Echo9573** [comment]
This question is similar to #770

**Comment by Echo9573** [comment]
Hi, @Yancey1989, When I manually convert the double type to a float, another exception is also thrown, as followings shows. Maybe it is necessary for SQLFlow to support common database types.

--------------------------------Exception------------------------------
_Rendezvous: <_Rendezvous of RPC that terminated with:
	status = StatusCode.UNKNOWN
	details = "genTF unsupported label data type: BIGINT_TYPE"
	debug_error_string = "{"created":"@1573389345.693687540","description":"Error received from peer ipv4:10.82.128.7:8005","file":"src/core/lib/surface/call.cc","file_line":1055,"grpc_message":"genTF unsupported label data type: BIGINT_TYPE","grpc_status":2}"
>

**Comment by Echo9573** [comment]
The hive data type is as follows:
Tinyint, smallint, int, bigint, boolean, float, double, string, binary, timestamp, decimal, date, varchar, char, array, map, struct.
Common data types may have at least tinyint, smallint, int, bigint, float, double, decimal, varchar, char

**Comment by Yancey0623** [comment]
Hi @Echo9573 , thanks for your report, I will reproduce your case.

-------

> The hive data type is as follows:
Tinyint, smallint, int, bigint, boolean, float, double, string, binary, timestamp, decimal, date, varchar, char, array, map, struct.
Common data types may have at least tinyint, smallint, int, bigint, float, double, decimal, varchar, char

It seems that `array/map/struct` is more complex, can you show some cases that how to use them in Tensorflow/XGBoost or other toolkit ?
